WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewView.swift

```diff
@@ -168,16 +151,9 @@ private extension WooPaymentsDepositsCurrencyOverviewView {
             "Pending funds",
             comment: "Title for pending funds overview in WooPayments Deposits view. " +
             "This shows the balance which will be made available for pay out later.")
-        static let depositsHeader = NSLocalizedString(
-            "Deposits",
-            comment: "Section header for the deposits list in the WooPayments Deposits overview")
-        static let nextDepositRowTitle = NSLocalizedString(
-            "Next",
-            comment: "Row title for the next deposit in the WooPayments Deposits overview")
-        static let lastDepositRowTitle = NSLocalizedString(
-            "deposits.currency.overview.depositTable.last.title",
-            value: "Last",
-            comment: "Row title for the last (previous) deposit in the WooPayments Deposits overview")
+        static let lastDepositHeader = NSLocalizedString(
+            "Last Deposit",
+            comment: "Section header for the last deposit in the WooPayments Deposits overview")
```

Comment:
Please follow the new style for localized strings by adding a key to the string.

Reply:
Addressed here: https://github.com/woocommerce/woocommerce-ios/pull/11533/commits/c47d5d7efc5a00ad8fbfe6ff8fbbd739de011719